src/recap

```diff
@@ -252,7 +252,7 @@ plugins_info(){
 plugins_load() {
   local plugin_dir="$1"
   log INFO "Loading plugins from: ${plugin_dir}"
-  for plugin in "${plugin_dir}"/*; do
+  for plugin in $( ls -1 "${plugin_dir}" ); do
     log INFO "Loading plugin: ${plugin}"
     source ${plugin}
```

Comment:
It seems the `plugin_dir` is missing from the sourced file and ideally it should be quoted:
`source "${plugin_dir%/}/${plugin}"`
or the following if you want to get rid of all the possible extra slashes _(I know they don't hurt)_ in the path (eg: `/tmp/plugin//file`):
```
plugin_file=$(realpath -m "${plugin_dir}/${plugin}")
source "${plugin_file}"
```